roles/grafana/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+graphite_listen: 8080
```

Comment:
Maybe `graphite_listen_port`?  Or possibly just use `graphite_port`, which means if someone overrides it for the graphite install it would automatically be correct here...?